### ISSUE TITLE
Search results page shows results when no search term is entered

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#1192]
**Please complete the following information** 
**Describe the bug**
A clear and concise description of what the bug is:
Even when no search term is entered before clicking the search button, search results show (314 so it could possibly be that it's just returning every single page as a result).

There are some accessibility issues with this but in the first instance we need to decide if we want any results to show at all

**To Reproduce**
1. Go to 'https://sdgdata.gov.uk/'
2. Click on the search button - don't enter a search term
3. Scroll down to under search bar

**Expected behavior**
A clear and concise description of what you expected to happen:
Either:

- no results show (with message)
- accessibility issues are resolved

**Screenshots**
![image](https://user-images.githubusercontent.com/43404358/117292464-24132b80-ae68-11eb-87cb-50831b9ae7a4.png)

**Accessibility issues**

1. If a user clicks the ‘search’ button without entering any search text, the main page content appears to be hidden from assistive technology. None of the content is relayed with screen reading software, and none of the links are recognised with voice activation software. Instead, the ‘loading search results graphic’ is identified with screen reading software, despite appearing to be hidden. 
  Solution: Ensure that information and functionality is presented to assistive technology. 
2. A search result for Pages >> /indicators.json is returned but it does not have a link or any description - this should be removed.

<img src="https://user-images.githubusercontent.com/43404358/117293267-14e0ad80-ae69-11eb-8531-89bc84ce139f.png" width="25%">

**Comment by phillipgardiner** [comment]
@brockfanning best practice is to not display any search results. I would therefore promote if a user presses search (either on the homepage or main search page) without entering anything, we display the 'No results found' page with an additional sentence below saying 'Try adjusting your search to find what your looking for' - this sentence will help guide them for next steps.

Hope that makes sense, happy to mock something up if required.
